features/step_definitions/logging.rb

```diff
@@ -705,3 +705,37 @@
     end
   end
 end
+
+# Delete the fluentd buffer files in nodes. the fluentd must be deleted before running this step.
+Given /^logging collector bufferfiles are cleared on all nodes$/ do
+  ensure_admin_tagged
+  serviceaccount="collector-clean"
+  ds_name="collector-clean"
+  user(0)
+  step %Q/I have a project/
+
+  fluentd_pods = BushSlicer::Pod.get_labeled("logging-infra=fluentd", project: project("openshift-logging", switch: false), user: admin)
+  if(fluentd_pods.length()>0)
+    puts("The fluetnd pods is still running")
+    @result[":success"] = false
+  else
+    sa = service_account("#{serviceaccount}")
+    @result =  sa.create(by: admin)
+    step %Q/the step should succeed/
+
+    _opts = {scc: "privileged", user_name: "system:serviceaccount:#{project.name}:#{serviceaccount}"}
+    @result = admin.cli_exec(:oadm_policy_add_scc_to_user, **_opts)
+    step %Q/the step should succeed/
```

Comment:
Here should use something like:
```raise "some error messages" unless @result[:success]```

Reply:
I think ' step %Q/the step should succeed/' should be the similar with 'raise "some error messages" unless @result[:success]'. Of course, Raise can provide more clear message.